data/json/recipes/recipe_food.json

```
@@ -4359,7 +4383,14 @@
     "components": [
       [ [ "eggs_bird", 2, "LIST" ], [ "egg_reptile", 2 ] ],
       [ [ "any_butter_or_oil", 2, "LIST" ] ],
-      [ [ "mushroom", 1 ], [ "mushroom_morel", 1 ], [ "garlic_clove", 1 ], [ "onion", 1 ] ],
+      [
+        [ "mushroom", 1 ],
+        [ "mushroom_morel", 1 ],
+        [ "garlic_clove", 1 ],
+        [ "wild_garlic", 1 ],
+        [ "onion", 1 ],
+        [ "groundnut", 1 ]
```

I'd consider using either `groundnut_prepared` or `veggy_potatolike` here.

The first requires the ground nuts be peeled and lightly boiled (to destroy protease inhibitors), the second does the same but also allows real potatoes to be used.

```suggestion
        [ "veggy_potatolike", 1, "LIST" ]
```